<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormVActVacuum</class>
 <widget class="QWidget" name="FormVActVacuum">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>815</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>815</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>758</width>
    <height>815</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>-1</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string notr="true">扩展输出</string>
  </property>
  <property name="toolTip">
   <string notr="true"/>
  </property>
  <property name="statusTip">
   <string notr="true"/>
  </property>
  <property name="whatsThis">
   <string notr="true"/>
  </property>
  <property name="accessibleName">
   <string notr="true"/>
  </property>
  <property name="accessibleDescription">
   <string notr="true"/>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 28px &quot;微软雅黑&quot;;</string>
  </property>
  <property name="windowFilePath">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="labelParaTitle">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel#labelParaTitle
{
	color: black;
	border-image: url(:/img/widget/Panel.png) 0 4 0 4;
	border-left: 4px transparent;
	border-right: 4px transparent;
}
</string>
     </property>
     <property name="text">
      <string>治具参数</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameExtOut">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="margin">
       <number>5</number>
      </property>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="labelPort">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>-1</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>输出端口：</string>
          </property>
          <property name="buddy">
           <cstring>lineEdit_var</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="xLineEdit" name="lineEdit_var">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>220</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>220</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>-1</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="labelPort_in">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>-1</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>输入端口：</string>
          </property>
          <property name="buddy">
           <cstring>lineEdit_var</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="xLineEdit" name="lineEdit_var_in">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>220</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>220</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>-1</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="xRadioButton" name="radioButton_on">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>-1</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string>输出打开</string>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="xPushButton" name="BtnOn">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>-1</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string extracomment="扩展输出开关">打开</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="xRadioButton" name="radioButton_off">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>-1</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string>输出关闭</string>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="xPushButton" name="BtnOff">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>-1</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string extracomment="扩展输出开关">关闭</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="labelDelay">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>-1</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>动作时间：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="xDoubleSpinBox" name="BoxDelay">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>220</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>220</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>-1</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="suffix">
           <string notr="true">s</string>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="maximum">
           <double>99.900000000000006</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="HasBtn" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="labelInterval">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>-1</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>执行间隔：</string>
          </property>
          <property name="buddy">
           <cstring>lineEdit_var</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="xSpinBox" name="BoxInterval">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>220</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>220</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>-1</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="suffix">
           <string notr="true"/>
          </property>
          <property name="prefix">
           <string notr="true"/>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>999</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="HasBtn" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="6" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="labelOutputCounter">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>-1</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>输出计数器：</string>
          </property>
          <property name="buddy">
           <cstring>lineEdit_var</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="xLineEdit" name="lineEditOutputCounter">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>220</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>220</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>-1</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="7" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="labelInputCounter">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>-1</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>输入计数器：</string>
          </property>
          <property name="buddy">
           <cstring>lineEdit_var</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="xLineEdit" name="lineEditInputCounter">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>220</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>220</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>-1</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="8" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>286</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>xPushButton</class>
   <extends>QPushButton</extends>
   <header>xpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>xSpinBox</class>
   <extends>QSpinBox</extends>
   <header>xspinbox.h</header>
   <slots>
    <signal>clicked()</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>xRadioButton</class>
   <extends>QRadioButton</extends>
   <header>xradiobutton.h</header>
  </customwidget>
  <customwidget>
   <class>xDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>xdoublespinbox.h</header>
   <slots>
    <signal>clicked()</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>xLineEdit</class>
   <extends>QLineEdit</extends>
   <header>xlineedit.h</header>
   <slots>
    <signal>clicked()</signal>
    <slot>inputIP()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>BoxDelay</sender>
   <signal>clicked()</signal>
   <receiver>FormVActVacuum</receiver>
   <slot>inputDelay()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>282</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>394</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BoxInterval</sender>
   <signal>clicked()</signal>
   <receiver>FormVActVacuum</receiver>
   <slot>inputInterval()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>675</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>394</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_var</sender>
   <signal>clicked()</signal>
   <receiver>FormVActVacuum</receiver>
   <slot>selectPort()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>282</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>394</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_var_in</sender>
   <signal>clicked()</signal>
   <receiver>FormVActVacuum</receiver>
   <slot>selectPortIn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>675</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>394</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>inputDelay()</slot>
  <slot>inputInterval()</slot>
  <slot>selectPort()</slot>
  <slot>extOutOn()</slot>
  <slot>extOutOff()</slot>
  <slot>selectPortIn()</slot>
 </slots>
</ui>
